src/cromshell/utilities/cromshellconfig.py

```diff
@@ -22,7 +22,8 @@
 cromwell_api = None
 # Defaults for variables will be set after functions have been defined
 config_dir = None
-submission_file = None
+submission_file_name = "all.workflow.database.tsv"
```

Comment:
Unsure if it's a pythonic convention, but typically for constants/macros, let's use uppercase letters to indicate it's a constant.

BTW, is it something intended to be configurable?